jerry-core/parser/js/byte-code.h

```diff
@@ -637,6 +637,9 @@ typedef struct
   uint16_t ident_end;               /**< end position of the identifier group */
   uint16_t const_literal_end;       /**< end position of the const literal group */
   uint16_t literal_end;             /**< end position of the literal group */
+#ifdef JERRY_CPOINTER_32_BIT
+  uint16_t padding;                 /**< an unused value */
```

Comment:
Does that mean that if `JERRY_CPOINTER_32_BIT` then the size of both `cbc_uint8_arguments_t` and `cbc_uint16_arguments_t` **must** be a multiple of 4 bytes? I think that would deserve a static assert. When looking at `cbc_uint16_arguments_t`, it is not that obvious that the current size is 18, because `ecma_compiled_code_t` (the type of the first `header` field of this struct) is defined "far away", i.e., in a different include in a different subdir. If that type changes for whatever reason, then the current padding will need to be changed, and perhaps even `cbc_uint8_arguments_t` will require some padding.